docs/source/en/_toctree.yml

```diff
@@ -247,6 +247,8 @@
         title: ERNIE
```

Comment:
Not sure why there are the four changes above, something to revert probably.

Reply:
Thanks for spotting the nit! Should be fixed !